grails-plugin-controllers/src/main/groovy/org/grails/config/http/GrailsFilters.java

```diff
@@ -0,0 +1,48 @@
+package org.grails.config.http;
+
+/* Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.springframework.boot.autoconfigure.security.SecurityProperties;
+
+/**
+ * Stores the default order numbers of all Grails filters for use in configuration.
+ * These filters are run prior to the Spring Security Filter Chain which is at DEFAULT_FILTER_ORDER
```

Comment:
Do you want to add `@since` and `@author`?

Reply:
added since. don't care about author